src/gluonts/evaluation/_base.py

```diff
@@ -340,13 +372,23 @@ def get_aggregate_metrics(
             "OWA": "mean",
             "MSIS": "mean",
         }
+
+        if self.custom_eval_fn is not None:
+            for k, (_, agg_type, _) in self.custom_eval_fn.items():
+                try:
+                    custom_agg = {k: agg_type}
```

Comment:
I'm not sure I understand the need for the `try` block here, what could go wrong?